TrueCraft.Core/Logic/Blocks/LeavesBlock.cs

```diff
@@ -33,6 +56,24 @@ public class LeavesBlock : BlockProvider
             return new Tuple<int, int>(4, 3);
         }
 
+        public void TryDecay(Coordinates3D coords, IMultiplayerServer server, IWorld world){
+            foreach (Coordinates3D a in Adjacent)
+            {
+                var c = a + coords;
+                var id = world.GetBlockID(c);
+                if (id == WoodBlock.BlockID)
+                {
+                    world.SetMetadata(coords, 0x0);
```

Comment:
This will change the leaf type (i.e. birch, spruce, pine). Just change the one bit.